crypto/build.info

```diff
@@ -104,6 +104,7 @@ SOURCE[../libcrypto]=$UTIL_COMMON \
         o_fopen.c getenv.c o_init.c init.c trace.c provider.c provider_child.c \
         punycode.c passphrase.c
 SOURCE[../providers/libfips.a]=$UTIL_COMMON
+SOURCE[../providers/liblegacy.a]=$UTIL_COMMON
```

Comment:
???  The legacy provider is linked with libcrypto anyway, and I think there are platforms where this will become a problem, possibly when `no-shared` is configured.

Reply:
I've since removed this.  I don't know what's causing the errors yet.

Reply:
And yeah, I understand that you get problems with unexported symbols...  The other way of handling this is to add `param_trie.c` to the liblegacy.a source separately

Reply:
Trying that now.

Reply:
Hmmm, that didn't appear to do anything.
It's late, I'm tired ...